components/centraldashboard/package-lock.json

```diff
@@ -1,8 +1,17583 @@
 {
```

Comment:
I have the feeling that the changes on this file are so many because I've been using node v18.0.0 locally. I'll check